### PR TITLE
recipe for MA

### PR DESCRIPTION
**Pull request by ITBE-Lab**
:information_source:
If this is your first pull request to Bioconda please ping `@bioconda/core` so it can be reviewed and to request being added as a member of the Bioconda team. Members of the Bioconda team are able to merge their own pull requests once tests and linting have passed.

----------------

* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [X] This PR adds a new recipe.
* [X] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/ma/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{% set version = "1.1.1" %}
+{% set sha256 = "a09a8b3d1c6281e2a4c9cddaa13b7ca7e8c600796320df6de3419facaa4d652a" %}
+package:
+  name: ma
+  version: '{{version}}'
+source:
+  url: "https://github.com/ITBE-Lab/MA/archive/{{version}}.tar.gz"
+  sha256: "{{sha256}}"
+build:
+  number: 1
```
Comment (micknudsen) [inline recipes/ma/meta.yaml]: Since this is the first build, the build number should be set to `0`.

Comment (micknudsen) [reply recipes/ma/meta.yaml]: Also, only the Mac OS build fails. If this package is not intended for Mac OS, you can skip it by adding `skip: true  # [osx]` to the build section.

Comment (ITBE-Lab) [reply recipes/ma/meta.yaml]: Thank you!
I have changed the build number and added skip: true # [osx].

**Comment by dpryan79** [comment]
Thanks @ITBE-Lab! Let me know if you'd like to be added to bioconda.

**Comment by ITBE-Lab** [comment]
Thanks. It would be nice, if you add this account to bioconda.

**Pull request merged**